filters/include/pcl/filters/experimental/functor_filter.h

```diff
@@ -96,5 +96,12 @@ class FunctorFilter : public FilterIndices<PointT> {
     }
   }
 };
+
+template <class PointT>
+using SimpleFilterFunction =
+    std::function<bool(const pcl::PointCloud<PointT>&, pcl::index_t)>;
+
+template <class PointT>
+using FunctionFilter = FunctorFilter<PointT, SimpleFilterFunction<PointT>>;
```

Comment:
Oh, `FunctionFilter` is actually different from `FunctorFilter`. Maybe there is a better name to avoid confusion?

Reply:
FunctorFilter is generic, and I wish to not expose it directly, at least till CTAD in C++17 lands for PCL. Should I create a namespace `advanced` and put the impl in that and expose just `FunctionFilter`?